src/styles/blocks/_page.scss

```diff
@@ -4,6 +4,8 @@
   scroll-behavior: smooth;
   box-sizing: border-box;
   overflow-x: hidden;
+  max-width: 100vw;
+  scroll-behavior: smooth;
```

Comment:
add transition on buttons and clickable elements
<img width="637" alt="image" src="https://user-images.githubusercontent.com/66749919/221357825-ffaa1f12-ac78-44e5-a71a-a5e291600f71.png">
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/66749919/221357891-f1357a04-cc1e-46f7-b3d8-85a9623e0ecf.png">
<img width="396" alt="image" src="https://user-images.githubusercontent.com/66749919/221357913-a787fb3e-caad-4358-ab96-cb40b693058e.png">
...